changelog.md

```diff
@@ -1,8 +1,97 @@
-## Changes from 1.1.0 to (unreleased version)
+## Changes from 1.2.0 to (unreleased version)
+
+
+## Changes from 1.1.0 to 1.2.0
+
+### Recommended Mesos version is 1.0.0
+
+### Breaking Changes
+
+#### You need Mesos 1.0.0 or higher
+Starting with Marathon 1.2.0, Mesos 1.0.0 or later is required.
+
+#### New leader election library
+The leader election code has been greatly improved and is based on [Curator](http://curator.apache.org),a well-known, well-tested library. The new leader election functionality is now more robust.
+
+__Caution: the new leader election library is not compatible with Marathon versions prior to 1.2.0__
+To upgrade to this version, stop all older Marathon instances before the new version starts.
+Otherwise, there is a risk of more than one leading master.
+
+
+#### Framework authentication command line flags
+Prior versions of Marathon have tried to authenticate whenever a principal has been provided via the command line.
+Framework authentication is now explicit. There is a command line toggle option for authentication: `--mesos_authentication`.
+This toggle is disabled by default. You must now supply this flag to use framework authentication.
+
+
```

Comment:
We fixed the `unique service ports` validation. Even though we fixed a bug, it could be seen as a breaking change.